bottlecap/src/traces/stats_flusher.rs

```diff
@@ -29,6 +29,7 @@ pub trait StatsFlusher {
 pub struct ServerlessStatsFlusher {
     pub buffer: Arc<Mutex<Vec<pb::ClientStatsPayload>>>,
     pub config: Arc<config::Config>,
+    pub resolved_api_key: String,
```

Comment:
Shouldn't we just name this field `api_key` and then pass the resolved one?

Reply:
The resolved_api_key is the name we used elsewhere. config also has an API key field which I think makes it a little confusing, so I chose this name.

We are already passing the resolved one, see the changes in main.